docs/developer-center/guides/02-Context.md

```diff
@@ -0,0 +1,75 @@
+## Context
+
+Before we can do anything with CARTOframes, we need to authenticate against a CARTO account by passing in CARTO credentials. You will need your username (`base_url`) and an API key (`api_key`), which can be found at **http://your_user_name.carto.com/your_apps.**
+
+![API Key - CARTO Dashboard](../../img/guides/context/api-keys.png)
+
+* `base_url`: which is the URL of your CARTO account (`https://your_user_name.carto.com/`)
+* `api_key`: if the dataset is **public**, we can use `default_public`. Otherwise, we need to set the API key.
+
+There are two ways to use our credentials:
+
+1. Setting the same credentials by default, which is called the **Default Context**
+2. Creating different contexts and passing them to the Map, Layer or Source we want to create, by using the `Context` class.
+
+The elements needed to create contexts are under the `cartoframes.auth` namespace.
+
+### Default Context
+
+With `set_default_context`, the same context will be used by all the layers and sources by default.
+
+```py
+from cartoframes.auth import set_default_context
+
+set_default_context(
```

Comment:
`set_default_context` allows this `set_default_context(https://your_user_name.carto.com/)` for public datasets. I would comment this in the guides